bindings/nodejs/lib/types/client/burn.ts

```diff
@@ -12,5 +12,5 @@ export interface Burn {
     /** Foundries to burn */
     foundries?: string[];
     /** Amounts of native tokens to burn */
-    nativeTokens?: INativeToken[];
+    nativeTokens?: Map<string, HexEncodedAmount>;
```

Comment:
Why is it not working with the interface?

Reply:
Because nativeTokens is a `BTreeMap` on the Rust side and with an array we get this error then `'invalid type: sequence, expected a map at line 1 column 112'`